ui/src/common/ContextMenus.js

```diff
@@ -130,12 +130,14 @@ const ContextMenu = ({
 
   return (
     <span className={clsx(classes.noWrap, className)}>
-      <LoveButton
-        record={record}
-        resource={resource}
-        visible={showLove}
-        color={color}
-      />
+      {config.enableFavourites && (
+        <LoveButton
+          record={record}
+          resource={resource}
+          visible={showLove}
```

Comment:
To not show the `LoveButton`, you should set its `visible` attr to `false`, so it still occupies the space in the screen, but is not shown. This avoids breaking the layout (as it is happening in some of your screenshots)